stories/Components/Forms/Dropdowns/Multiselect/MultiSelect.jsx

```diff
@@ -17,52 +18,155 @@ const SelectTag = ({ text, eleId }) => {
-          <Checkbox label={`${text}`} value="category5" id={`category5-${eleId}`} />
-        </li>
-      </ul>
+      {args.variant === "Checkbox" && (
+        <ul
+          className={[
+            "options",
+            args.Height === "Fix height" ? " fix-height" : "",
+          ].join("")}
+          data-type="region"
+          aria-labelledby={`select_button-${eleId}`}
+          role="listbox"
+          aria-multiselectable="true"
+          aria-hidden="true"
+          aria-modal="true"
+        >
+          <li role="option">
+            <Checkbox
+              label={`${text}`}
+              value="category1"
+              id={`category1-${eleId}`}
+            />
+          </li>
+          <li role="option" className="has-submenu">
+            <button className="has-submenu__button checkbox-item">
+              {text}
+            </button>
+            <ul role="listbox" className="sub-menu">
+              <li role="option">
+                <Checkbox
+                  label={`${text}`}
+                  value="subcategory1"
+                  id={`subcategory1-${eleId}`}
+                />
+              </li>
+              <li role="option">
+                <Checkbox
+                  label={`${text}`}
+                  value="subcategory2"
+                  id={`subcategory2-${eleId}`}
+                />
+              </li>
+            </ul>
+          </li>
+          <li role="option">
+            <Checkbox
+              label={`${text}`}
+              value="category2"
+              id={`category2-${eleId}`}
+            />
+          </li>
+          <li role="option">
+            <Checkbox
+              label={`${text}`}
+              value="category3"
+              id={`category3-${eleId}`}
+            />
+          </li>
+          <li role="option">
+            <Checkbox
+              label={`${text}`}
+              value="category4"
+              id={`category4-${eleId}`}
+            />
+          </li>
+          <li role="option">
+            <Checkbox
+              label={`${text}`}
+              value="category5"
+              id={`category5-${eleId}`}
+            />
+          </li>
+        </ul>
+      )}
+      {args.variant === "Radio" && (
+        <ul
+          className={[
+            "options",
+            args.Height === "Fix height" ? " fix-height" : "",
+          ].join("")}
+          data-type="region"
+          aria-labelledby={`select_button-${eleId}`}
+          role="listbox"
+          aria-multiselectable="true"
+          aria-hidden="true"
+          aria-modal="true"
+        >
+          <li role="option">
+            <Radio
+              label={`${text}`}
+              value="category1"
+              id={`category1-${eleId}`}
+              name={`category1-${eleId}`}
+            />
+          </li>
+          <li role="option" className="has-submenu">
+            <button className="has-submenu__button checkbox-item">
```

Comment:
@Ashish-srij Kindly remove class `has-submenu__button` and use parent class to style this element.

Reply:
@surbhi90 Done.